src/summary/VisualizerManager.jsx

```diff
@@ -367,4 +371,25 @@ export default class VisualizerManager {
             </svg>
         );
     }
+
+    _longitudinalViewIcon = (isSelected) => {
+        const strokeColor = isSelected ? "#3F3F3F" : "#CCCCCC";
+        const fillColor = isSelected ? "#8d9196" : "#C4C4C4";
```

Comment:
I would make these the same as the stroke color for consistency

Reply:
resolved